jumpcloud/resource_user_group_membership.go

```diff
@@ -85,8 +85,14 @@ func resourceUserGroupMembershipRead(d *schema.ResourceData, m interface{}) erro
 	config := m.(*jcapiv2.Configuration)
 	client := jcapiv2.NewAPIClient(config)
 
+	var limit int32 = 100
+	optionals := map[string]interface{}{
+		"groupId": d.Get("groupid").(string),
+		"limit":   limit,
```

Comment:
How about just `"limit":   100,` ?

Reply:
That did not work for me as the "100" was created as a var of type uint , resulting in a run-time error. 
I did not find a way to cast it in-line.

Reply:
`"limit": int32(100),` ?

Reply:
Thanks - That worked.